surveys/partisanFeelingThermometerUS/partisanFeelingThermometerUS.score.js

```diff
@@ -1,4 +1,20 @@
 export default function scoreFunc(responses) {
-  const result = {};
+  const minScore = 0;
+  const maxScore = 100;
+
+  const normRepublicanTemp = (parseInt(responses["republicanTemp"])
+    - minScore) / (maxScore - minScore);
+  const normDemocratTemp = (parseInt(responses["democratTemp"])
+    - minScore) / (maxScore - minScore);
+
+  const order = Object.keys(responses);
+
+  const result = {
+    normRepublicanTemp: normRepublicanTemp,
```

Comment:
can you add some comments here and in the associated readme describing what the different scores mean?